static/app/gettingStartedDocs/javascript/solidstart.tsx

```diff
@@ -0,0 +1,481 @@
+            {codeFile: <code />, codeFolder: <code />}
+          ),
+        },
+        ...(params.isPerformanceSelected
+          ? [
+              {
+                description: tct(
+                  'Complete the setup by adding the Sentry middleware to your [code:src/middleware.ts] file',
+                  {code: <code />}
+                ),
+                code: [
+                  {
+                    label: 'TypeScript',
+                    // value and language are in JS to get consistent syntax highlighting
+                    // we aren't using any Typescript specific code in these snippets but
+                    // want a typescript ending.
+                    value: 'javascript',
+                    language: 'javascript',
+                    code: getSdkMiddlewareSetup(),
+                  },
+                ],
+              },
+              {
+                description: tct(
+                  "If you're using [solidRouterLink:Solid Router], wrap your [codeRouter:Router] with [codeRouterWrapping:withSentryRouterRouting]. This creates a higher order component, which will enable Sentry to collect navigation spans.",
+                  {
+                    codeRouter: <code />,
+                    codeRouterWrapping: <code />,
+                    solidRouterLink: (
+                      <ExternalLink href="https://docs.solidjs.com/solid-router" />
+                    ),
+                  }
+                ),
+                code: [
+                  {
+                    label: 'TypeScript',
+                    value: 'typescript',
+                    language: 'typescript',
+                    code: getSdkRouterWrappingSetup(),
+                  },
+                ],
+              },
+            ]
+          : []),
+        ...(params.isProfilingSelected
+          ? [getProfilingDocumentHeaderConfigurationStep()]
+          : []),
+        {
+          description: tct(
+            'Add an [codeFlag:--import] flag to the [codeNodeOptions:NODE_OPTIONS] environment variable wherever you run your application to import [codeInstrument:public/instrument.server.mjs]. For example, update your [codeScripts:scripts] entry in [codePackageJson:package.json]',
+            {
+              codeFlag: <code />,
+              codeNodeOptions: <code />,
+              codeInstrument: <code />,
+              codeScripts: <code />,
+              codePackageJson: <code />,
+            }
+          ),
+          code: [
+            {
+              label: 'JSON',
+              value: 'json',
+              language: 'json',
+              code: getSdkRun(),
+            },
+          ],
+        },
+      ],
+    },
+    {
+      title: t('Upload Source Maps'),
+      description: tct(
+        'To upload source maps to Sentry, follow the [link:instructions in our documentation].',
```

Comment:
Does it support the sentry wizard?
Then there would be a util that generates this step:
[getUploadSourceMapStep](https://github.com/getsentry/sentry/blob/master/static/app/components/onboarding/gettingStartedDoc/utils/index.tsx)